melodies_monet/util/cal_mod_no2col.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2022 National Center for Atmospheric Research and National Oceanic and Atmospheric Administration
+# SPDX-License-Identifier: Apache-2.0
+
+# calculate WRF-Chem NO2 trop. columns, for further pair with satellite swath data
```

Comment:
Is this only for WRF comparisons right now and you plan to generalize to more models later? Or should we call this python script cal_wrf_no2col.py?

Reply:
This is for WRF right now. I'm not sure if this can be applied to other models. It should work but with several modifications such as variable names, etc. I think it does make sense to change it to cal_wrf_no2col.py.